### DISCUSSION TITLE
allow Energy dashboard to Not be listed in Side panel

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#11455] [Dashboard]
as per discission in #beta Discord https://discord.com/channels/330944238910963714/427516175237382144/936216698661531669

I would like to suggest to allow us to not to show the dashboard , and if so desired use the energy cards in our Views.

https://www.home-assistant.io/lovelace/energy/

<img width="1667" alt="Schermafbeelding 2022-01-27 om 13 20 00" src="https://user-images.githubusercontent.com/33354141/151357701-b5079e07-ada7-4cd0-89f2-458ad7268ddd.png">

**Comment by Mariusthvdb** [comment]
partnering that, please also consider allowing the option to have `require_admin: true/false` for the energy dashboard. It is not necessary for all home users to see this.

**Comment by Santanachia** [comment]
You can do this by pressing and holding the text above the menu:
![image](https://user-images.githubusercontent.com/4628507/195581078-9fb907d6-9d4c-4686-a514-ca9a1afa621d.png)
![image](https://user-images.githubusercontent.com/4628507/195581114-c0719399-f15a-4a42-85e8-16a8461fe783.png)

**Comment by Mariusthvdb** [comment]
Sure, but, that's hardly what I am after. Its a per user, per browser, per etc etc setting.
As an Admin I just want to be able to not show this point blanc.Preferably allow it to special users. as in opening topic
